packages/ketcher-core/src/application/editor/tools/Bond.ts

```diff
@@ -465,21 +466,36 @@ class PolymerBond implements BaseTool {
 
     const atomRenderer = event.target.__data__ as AtomRenderer;
     const monomer = this.bondRenderer?.polymerBond.firstMonomer;
+
+    if (
+      monomer instanceof Chem &&
```

Comment:
Please delete ```monomer instanceof Chem &&``` because restriction should work for all types of monomers

Reply:
<img width="885" alt="image" src="https://github.com/user-attachments/assets/c20b7933-3474-4075-87de-1944a6595e56" />

Reply:
I saw the description.
I'm 100% sure that it should work for every type of monomer. Discussed with @AlexeyGirin one more time, he approved. Please remove the line.